components/gitpod-protocol/src/frontend-dashboard-service.ts

```diff
@@ -46,6 +46,7 @@ export namespace IDEFrontendDashboardService {
 
         workspaceDescription: string;
         workspaceType: string;
+        credentialsToken: string;
```

Comment:
@mustard-mh I don't have a good overview over the structure of the new IDE service. But from the past I can tell that mixing "Status" and "token" always ends up problematic. What about introducing a new method "getCredentialsToken" (or similar) on the interface (I cannot tell which is which just by the names "Server" and "Client")?

Reply:
Maybe it's a naming problem as `Status` can be `Info` or anything else, it's a define of interface that `supervisor/frontend` is needed. 

🙏 I will update PR to change the name with compatibility (first we will keep them (old and new name) both, but will remove it later)

Reply:
[Addressed](https://github.com/gitpod-io/gitpod/compare/fa01d4504e464b05a8a3df5e55ac7f62186bfff9..913fa7145fa1f591022c3ec6d9b5d8d92782df00), going to do compatibility test in another PR, will post results here and unhold this PR if it works